PFW-1435 Nozzle change menu

3d-gussner opened the pull request:
Added basic nozzle change menu.

The `prusa.io/nozzle-mk3s` is 20 chars long and the message above should not exceed c=20 r=2 as the last char on LCD page is a :white_check_mark: and will cover the `s` of the `nozzle-mk3s` link if the message uses a 3rd row.

Shorten the messages due to limited space for translations. 

Odd issue:
- [x] Switching sometimes hangs until a Gcode is send via serial line. @leptun Do you have an idea how that is happening?

ToDo:
- [x] Translations:
- [ ] French is at its limit and can't be translated.

3d-gussner committed:
Nozzle change menu

leptun committed:
Fix recursion

wavexx commented on Firmware/ultralcd.cpp:
```diff
@@ -1005,6 +1006,57 @@ void lcd_commands()
+                lcd_show_fullscreen_message_and_wait_P(_T(MSG_NOZZLE_CNG_READ_HELP));
+                lcd_update_enabled = true;
+                lcd_draw_update = 2; //force lcd clear and update after the stack unwinds.
+                enquecommand_P(PSTR("G28W"));
```
I guess this should be `G28 W`?

Hauzman commented:
@3d-gussner 
Ro translation 
msgid "Hotend at 280C! Nozzle changed and tightened to specs?"
msgstr "Horend la 280C!Varf Schimbat si strans conf. specificatiei?"

msgid "Nozzle change"
msgstr "Schimbare Varf"

3d-gussner commented:
> @3d-gussner Ro translation msgid "Hotend at 280C! Nozzle changed and tightened to specs?" msgstr "Horend la 280C!Varf Schimbat si strans conf. specificatiei?"
> 
> msgid "Nozzle change" msgstr "Schimbare VArf"

Thanks I guess  "Schimbare Varf" right?

Hauzman commented:
> > @3d-gussner Ro translation msgid "Hotend at 280C! Nozzle changed and tightened to specs?" msgstr "Horend la 280C!Varf Schimbat si strans conf. specificatiei?"
> > 
> > msgid "Nozzle change" msgstr "Schimbare VArf"
> 
> Thanks I guess  "Schimbare Varf" right?

Yes sorry for the error

leptun committed:
fullscreen message: use \n to force the next word on the next line

Useful for printing short URLs since those contain punctuation characters, which get treated at the end of a word under usual circumstances

leptun commented:
`msgstr "Hotend la 280C!Varf Schimbat si strans conf. specificatiei?"`
Fixed a typo.

3d-gussner committed:
Update internal translations
Fix G28 W
